coalib/output/Interactions.py

```diff
@@ -10,6 +10,7 @@ def fail_acquire_settings(log_printer, settings_names_dict, section):
                                 a list containing a description in [0] and the
                                 name of the bears who need this setting in [1]
                                 and following.
+    :param section:             The section the action corresponds to.
```

Comment:
What "action"? :)

Reply:
It is deprecated so can I state that in the docstring . I cannot get a hold of its functionality and according to previous commits this was supposed to be its documentation.